internal/server/resource/resource.go

```diff
@@ -45,6 +46,7 @@ type Cache struct {
 	RawSvg                    map[string]*pb.StatVarGroupNode
 	SvgSearchIndex            *SearchIndex
 	BlocklistedSvgSearchIndex *SearchIndex
+	SearchIndex               bleve.Index
```

Comment:
Since there are other indexes already, might make this more precise, like BleveSearchIndex?